app/src/main/java/com/example/navadon/androidnamecard/MainActivity.java

```diff
@@ -1,13 +1,34 @@
 package com.example.navadon.androidnamecard;
 
-import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
 
 public class MainActivity extends AppCompatActivity {
-
+    Button edit_btn;
+    TextView phone_number;
+    TextView name;
+    TextView email;
+    TextView address;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        edit_btn = (Button) findViewById(R.id.edit_btn);
+        phone_number = (TextView) findViewById(R.id.phone_number);
+        name = (TextView) findViewById(R.id.name);
+        email = (TextView) findViewById(R.id.email);
+        address = (TextView) findViewById(R.id.address);
+        edit_btn.setOnClickListener(new View.OnClickListener() {
```

Comment:
You were asked to do it differently. 
If the summation of student IDs is odd, 
use “Option 4” from OnClickEvent
	(Adding View.OnClickListener as class attribute and selecting view via id)